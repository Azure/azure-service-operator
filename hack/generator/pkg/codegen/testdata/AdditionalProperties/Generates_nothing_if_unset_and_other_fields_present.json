{
    "$comment": "Here we check that if there are other fields present and 'additionalProperties' is not explicitly set then nothing additional is generated (this is against spec but see 'additionalProperties' comments in jsonast)",

    "id": "https://test.test/schemas/2020-01-01/test.json",
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "Test",
    "type": "object",
    "properties": {
        "tags": {
            "type": "object",
            "properties": {
                "otherField": {
                    "type": "string"
                }
            }
        }
    }
}