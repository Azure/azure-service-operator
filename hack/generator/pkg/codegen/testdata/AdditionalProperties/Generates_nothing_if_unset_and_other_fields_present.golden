// Code generated by k8s-infra. DO NOT EDIT.
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
package v1alpha1api20200101

//Generated from: https://test.test/schemas/2020-01-01/test.json
type Test struct {
	Tags *Test_Tags `json:"tags,omitempty"`
}

type Test_Tags struct {
	OtherField *string `json:"otherField,omitempty"`
}
